packages/nimble-components/CONTRIBUTING.md

```diff
@@ -332,6 +332,14 @@ const nimbleButton = Button.compose({
 });
 ```
 
+### Leverage mixins for shared APIs across components
```

Comment:
I think it would be really helpful if we could provide guidance of when to use one patter vs the other. Assuming we know that guidance, could you provide a few sentences giving that guidance?

Reply:
+1, In particular, at least to start I'd recommend the Constrained Mixin pattern for any mixins we author in Nimble as it requires less manual type manipulation but continuing to use the FAST `applyMixin` helper for leveraging Mixins already defined in FastFoundation.